src/lambda-calculus.js

```diff
@@ -391,6 +400,8 @@ function evalLC(term) {
           term = lastTerm;
           env = lastEnv;
         } else { // lastTerm is a JS function
+          if (!(lastTerm instanceof Function))
```

Comment:
additional information should be printed depending on `verbosity`.
`Calm` ( and therefore all verbosities ) should _only_ raise a bare `EvalError`.
Voile be damned.
If we decide to reengineer how we do this, we should do it everywhere. For now, we should be consistent.